website/pages/partials/provisioners/common-config.mdx

```diff
@@ -10,17 +10,33 @@ Parameters common to all provisioners:
 - `override` (object) - Override the builder with different settings for a
   specific builder, eg :
 
+  In JSON:
   ```json
   {
     "type": "shell",
     "script": "script.sh",
     "override": {
-      "vmware-iso": {
-        "execute_command": "echo 'password' | sudo -S bash {{.Path}}"
+      "example_one": {
```

Comment:
While we're updating this, it kind of makes sense to swap this out for a very small but fully functional build:

```json
{
	"builders": [
		{
			"type": "null",
			"name": "example1",
			"communicator": "none"
		},
		{
			"type": "null",
			"name": "example2",
			"communicator": "none"
		}
	],
	"provisioners": [
		{
			"type": "shell-local",
			"inline": ["echo not overridden"],
			"override": {
				"example2": {
					"inline": ["echo yes overridden"]
				}
			}
		}
	]
}
```

```hcl
source "null" "example1" {
    communicator = "none"
}

source "null" "example2" {
    communicator = "none"
}

build {
    sources = ["source.null.example1", "source.null.example2"]
    provisioner "shell-local" {
        inline = ["echo not overridden"]
        override = {
            example1 = {
                inline = ["echo yes overridden"]
            }
        }
    }
}
```

What do you think?

Reply:
And while I'm on it maybe these would be good files to add to build_test.go which doesn't seem to have any override tests?

Reply:
Both are great!! I liked better your examples, very simple!